java/client/src/main/java/glide/api/commands/BitmapBaseCommands.java

```diff
@@ -113,4 +114,24 @@ public interface BitmapBaseCommands {
      * }</pre>
      */
     CompletableFuture<Long> getbit(String key, long offset);
+
+    /**
+     * Perform a bitwise operation between multiple keys (containing string values) and store the
+     * result in the <code>destKey</code>.
+     *
+     * @see <a href="https://redis.io/commands/bitop/">redis.io</a> for details.
+     * @param bitwiseOperation The bitwise operation to perform.
+     * @param destKey The key that will store the resulting string.
+     * @param keys The list of keys to perform the bitwise operation on.
+     * @return The size of the string stored in <code>destKey</code>.
+     * @example
+     *     <pre>{@code
+     * client.set("key1", "A"); // "A" has binary value 01000001
+     * client.set("key2", "B"); // "B" has binary value 01000010
+     * Long payload = client.bitop(BitwiseOperation.AND, "destKey", new String[] {key1, key2}).get();
+     * assert "@" == client.get("destKey").get(); // "@" has binary value 01000000
+     * assert payload == 1L; // The size of the resulting string is 1.
+     * }</pre>
+     */
+    CompletableFuture<Long> bitop(BitwiseOperation bitwiseOperation, String destKey, String[] keys);
```

Comment:
Not sure how much this matters but it looks like all the other commands label the destination key variable as "destination", should we change destKey to be consistent? If so, we'll have to update the docs as well to reflect the rename
```suggestion
    CompletableFuture<Long> bitop(BitwiseOperation bitwiseOperation, String destination, String[] keys);
```

Reply:
That's a good question. The inconsistency stems from Redis itself as they switch between `destKey` and `destination`. For example, https://redis.io/docs/latest/commands/sinterstore/ vs. https://redis.io/docs/latest/commands/bitop/. What should we follow?

Reply:
upvote for `destination`

Reply:
`destinationKey` if it makes sense.  Don't be one of those people that shortens argument names just because they don't want to type it all out.

Reply:
`destKey` came from Redis docs. I changed it to `destination` to be more consistent in java though.